tests/test_wsgiref_inputwrapper_with_size.py

```diff
@@ -0,0 +1,32 @@
+import json
+
+import falcon
+from falcon import testing
+
+
+class TypeResource(testing.SimpleTestResource):
+    """ A simple resource to return the posted request body """
+    @falcon.before(testing.capture_responder_args)
+    def on_post(self, req, resp, **kwargs):
+        resp.status = falcon.HTTP_200
+        resp.body = json.dumps({'data': req.stream.read().decode('utf-8')})
```

Comment:
Might be worth adding an additional `#NOTE(masterkale): ...` comment here, to point out that no size is passed to `read()`, even though it is also mentioned in a comment later on.